src/pybites_search/all_content.py

```diff
@@ -0,0 +1,57 @@
+from rich.table import Table
```

Comment:
I like how you abstracted this in a new subclass which required you to follow the abc